vllm/sequence.py

```diff
@@ -1290,8 +1299,8 @@ class ExecuteModelRequest(
     finished_requests_ids: List[str] = msgspec.field(default_factory=list)
     # The last sampled token ids for multi step decoding.
     last_sampled_token_ids: Optional[torch.Tensor] = None
-    # Async postprocessor
-    output_proc_callback_fn: Optional[Callable] = None
+    # Async callback
+    async_callback: Optional[AsyncCallbackData] = None
```

Comment:
I feel this name is a bit generic? Maybe something like `request_output_callback` is better? i.e. to make it clearer what this is calling you back for/with

Reply:
I would prefer to have it async_callback because it is not only output_proc anymore in the async + multi-step PR. The callback starts to get more and more functionality inside.